dashboard/api_test.py

```diff
@@ -869,6 +979,13 @@ def test_past_course_runs(self):
         )
 
         result = api.get_user_program_info(self.user, self.edx_client)
+        # extract the right program from the result
+        program_result = None
```

Comment:
this integration test has 2 programs generated via the `setUpTestData` and the previous version of the test was assuming that the relevant one was the first in the result (and that is true) but there is no guarantee that this is always the case